### ISSUE TITLE
Questions about Training loss curve

### ISSUE THREAD

**Issue by bollossom** [guolinke/SphereAR#2]
The training loss for the generation task seems to be converging very slowly. After 80,000 steps, the loss is still around 1.4. To help me verify if my training is on the right track, would it be possible for you to share the loss curve from your own experiments about model SphereAR-B?

<img width="1098" height="682" alt="Image" src="https://github.com/user-attachments/assets/1fee6683-4913-4768-95b1-1e4536a1a59e" />

**Comment by hasaki321** [comment]
Hello, have you recently reproduced the relevant experiments in the paper?
